### DISCUSSION TITLE
/themes folder empty upon installation with Hugo module

### DISCUSSION THREAD

**Discussion by bob3x255** [jpanther/congo#467] [Q&A]
Hi! 

I am trying to install Congo v2 via Hugo module by following: https://jpanther.github.io/congo/docs/installation/#download-the-congo-theme
However, the theme folder is empty and the theme itself is not downloaded. 

Perhaps, there is a missing step?

Thanks!

**Comment by jpanther** [comment]
There is nothing created in the themes folder when installing Hugo modules, the theme will download to a hidden Hugo directory and build your site from there. This is not an issue though, as you do not need access to the theme files to build your site. You should see the the theme download when you initialise your Go project and run the server for the first time.

**Reply by bob3x255** [reply]
Thanks @jpanther.

In fact I do not see the theme download after I initialize the project and run `hugo server`. I may run `hugo mod vendor`, which will put all theme files in vendor folders but I don't think this is an intended way to do so.

**Reply by jpanther** [reply]
You are correct that vendoring is not needed for this process. Could you try running `hugo mod clean` and see if that helps?

**Reply by bob3x255** [reply]
tried `hugo mod clean` as well. It removed module, then I re-installed the theme via Hugo Module option, however, still same results for some reason. Note, this is my first time installing theme via Hugo Module. I just installed it via git submodule, works like a charm. Great theme!

**Reply by jpanther** [reply]
I do wonder if these sorts of issues are operating system related. There have been threads from time to time where certain people have trouble installing the Hugo modules but it's never clear what the root cause is.

**Reply by bob3x255** [reply]
Yeah, pure mystery to me. 
For the record I have: 
* macOS 13.0.1. 
* Go 1.19.4
* v0.108.0+extended

Just tried installing 2 other themes with Hugo Modules, same result - /themes folder empty. I also tried looking for a solution, however, couldn't find anything useful.
